doc/source/examples/incompressible-flow/3d-taylor-green-vortex/3d-taylor-green-vortex.rst

```diff
@@ -0,0 +1,374 @@
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np n_proc lethe-fluid tgv-matrix-based.prm
+
+and the matrix-free simulations can be launched by typing
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np n_proc lethe-fluid-matrix-free tgv-matrix-free.prm 
+
+For a 5 initial refinement (:math:`32^3` Q2 cells), the matrix-based solver takes around 1 hour and 20 minutes on 16 cores while the matrix-free solver takes less than 20 minutes. Running the same problem, but in Q3 (:math:`32^3` Q3 cells), the matri-free solver takes less than 2 hours while the matrix-based solver takes close to a day and consumes a tremendous amount of ram (approx. 80 GB). If you have 64 GB of ram, you can run an even finer mesh (:math:`64^3` Q3 cells) using the matrix-free solver in approximatively 16 hours.
+
+
+----------------------
+Results and Discussion
+----------------------
+
+The flow patterns generated by the Taylor-Green vortex are quite complex. The following animation displays the evolution of velcoity iso-contours as the vortex break downs and generate smaller structures.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------+
+| .. raw:: html                                                                                                                                      |
+|                                                                                                                                                    |
+|    <iframe width="520" height="400" src="https://www.youtube.com/embed/GGij2g_yz5g?si=Q81lcHyPd9ONxqT4"  frameborder="0" allowfullscreen></iframe> |
+|                                                                                                                                                    |
++----------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+Using the ``enstrophy.dat`` and ``kinetic_energy.dat`` files generated by Lethe, the temporal decay of the kinetic energy can be monitor. First, we calculate the time-derivative of the kinetic energy by invoking the script present in the example folder:
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 calculate_dissipation_rate.py -i kinetic_energy.dat -o output.dat
+
+Then, by invoking the script present in the example, a plot compairing the kinetic energy decay with the enstrophy is generated:
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 plot_dissipation_rate.py -ke kinetic_energy_decay.dat -ens enstrophy.dat -v 0.000625
+
+.. note:
+ 
+  A nice plot with a zoomed in section can be generated by adding the argument `-z True` to the command above.
```

Comment:
There is something wrong with this note because it does not appear in the documentation...

Reply:
Found the issue:
```suggestion
.. note::
 
  A nice plot with a zoomed in section can be generated by adding the argument ``-z True`` to the command above.
```